src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewClinicalDataBinMapper.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+
+<mapper  namespace="org.cbioportal.persistence.mybatisclickhouse.StudyViewMapper">
+    <sql id="sampleClinicalDataFromStudyViewFilter">
+        <where>
+            sample_unique_id IN (
+            <include refid="sampleUniqueIdsFromStudyViewFilter"/>
+            <if test="applyPatientIdFilters == true">
+                INTERSECT <include refid="getSampleIdsFromPatientIds"/>
+            </if>
+            )
+        </where>
+        <if test="attributeIds != null and !attributeIds.isEmpty()">
+            AND attribute_name IN
+            <foreach item="attributeId" collection="attributeIds" open="(" separator="," close=")">
+                #{attributeId}
+            </foreach>
+        </if>
+    </sql>
+
+    <sql id="patientClinicalDataFromStudyViewFilter">
+        <where>
+            patient_unique_id IN (
+                <include refid="getPatientIdsFromSampleIdFilters"/>
+                <if test="applyPatientIdFilters == true">
+                    INTERSECT <include refid="patientUniqueIdsFromStudyViewFilter"/>
+                </if>
+            )
+        </where>
+        <if test="attributeIds != null and !attributeIds.isEmpty()">
+            AND attribute_name IN
+            <foreach item="attributeId" collection="attributeIds" open="(" separator="," close=")">
+                #{attributeId}
+            </foreach>
+        </if>
+    </sql>
+
+    <select id="getPatientClinicalDataCountsForBinning">
+        SELECT
+            attribute_name as attributeId,
+            if(attribute_value='', 'NA', attribute_value) AS value,
+            count(value) as count
+        FROM clinical_data_derived
+        <include refid="patientClinicalDataFromStudyViewFilter" />
+        AND type = 'patient'
+        GROUP BY attribute_name, value
+    </select>
+
+    <select id="getSampleClinicalDataCountsForBinning">
+        SELECT
+            attribute_name as attributeId,
+            if(attribute_value='', 'NA', attribute_value) AS value,
+            count(value) as count
+        FROM clinical_data_derived
+        <include refid="sampleClinicalDataFromStudyViewFilter" />
+        AND type = 'sample'
+        GROUP BY attribute_name, value
+    </select>
```

Comment:
These queries are very similar to `getClinicalDataCounts` query. We may be able to directly use that one instead of adding new SQL.

https://github.com/cBioPortal/cbioportal/blob/b436b2b7b90e9fd72cc86d13d02abdffdf586008/src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewMapper.xml#L66-L74